ext/native/ext/cityhash/city.cpp

```diff
@@ -87,18 +88,19 @@ static uint32 UNALIGNED_LOAD32(const char *p) {
 #endif
 
 #else
+// TODO: Why is this path being built on Android? Slow.
```

Comment:
Yes comment should be removed. Also I have #9053 to fix all the ANDROID preprocessor defines to __ANDROID__.